source/openstack.rst

```diff
@@ -175,17 +175,62 @@ Pre-requisites
 
 .. note::
    EGI monitoring checks that your Keystone accepts clients with certificates
-   from the IGTF CAs. Please ensure that your server is correctly configured
+   from the IGTF CAs. Please ensure that your server is configured
    with the correct Certificate and Revocation path:
 
-   :: 
+   For Apache HTTPd
+     HTTPd is able to use CAs and CRLs contained in a directory
+     ::
 
       SSLCACertificatePath    /etc/grid-security/certificates
       SSLCARevocationPath     /etc/grid-security/certificates
 
-   IGTF CAs can be obtained from UMD, you can find repo files for your distribution
-   at http://repository.egi.eu/sw/production/cas/1/current/
-
+   For Nginx
```

Comment:
I think there is no easy way to run Keystone + oidc on NGINX, should we really include it?

Reply:
OK, I was understanding that some sites may have been using it, I agree that if it's not the case we can drop this and keep only haproxy doc.

Reply:
Done.